Bottom: 59e3468e6ee5cd293bc9f520aa1110168e32110a
Top:    62efdb1c2d8c08f98cd452f2ffc6283e3c7e3df2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 23:27:33 -0500

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
